app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,53 @@
+<header class="site-header">
+  <nav class="site-nav">
+    <div class="site-nav__action">
+      <%= link_to root_path do %>
+        <%= image_tag("bostonrb-logo.png") %><h2 class="site-nav-title">Boston Ruby Group</h2>
```

Comment:
This is an interesting CSS oddity. This line is intentionally written on one line because breaking this into to lines:
```
<%= image_tag("bostonrb-logo.png") %>
<h2 class="site-nav-title">Boston Ruby Group</h2>
```
produces a small line (see below). Yet, keeping this all on one line removes the small line. I'm open to suggestions if anyone else has seen this before :).
<img width="317" alt="screen shot 2018-02-08 at 10 35 23 pm" src="https://user-images.githubusercontent.com/4016985/36013394-1878aa20-0d21-11e8-901a-b7119063999a.png">

Reply:
I bet it's trying to make a line with the whitespace somehow. Maybe @vendelal has some ideas as to how to address this?

Reply:
When you have two inline-block elements next to each other it creates a small space, and I guess .erb files cause it to make a dash or something? If you add these properties to the `link`, it should fix the problem:

```
display: flex;
justify-content: center;
align-items: center;
text-decoration: none;
```